package-lock.json

```diff
@@ -1,8 +1,3229 @@
 {
   "name": "@cloudflare/kv-asset-handler",
-  "version": "0.1.1",
-  "lockfileVersion": 1,
+  "version": "0.1.2",
+  "lockfileVersion": 2,
```

Comment:
@signalnerve It looks like you regenerated this with a newer NPM version (7.x), thus the massive change to the structure here. I think this is fine, since it's backwards compatible, but just wanted to make sure this was known. 👍